spec.bs

```diff
@@ -8,6 +8,7 @@ Repository: WICG/shared-storage
 URL: https://github.com/WICG/shared-storage
 Editor: Camillia Smith Barnes, Google https://google.com, cammie@chromium.org
 Markup Shorthands: markdown yes
+Text Macro: PRIVATE-AGGREGATION-API [[private-aggregation-api|Private Aggregation API]]
```

Comment:
Wondering if it might be clearer to just put the full link in rather than defining a macro

Reply:
Fair enough! It's just that Bikeshed already knows the URL, so it feels unnecessarily verbose to repeat it.

I've seen other specs do something like this:

```md
Something something <cite>Private Aggregation API</cite>. [[!PRIVATE-AGGREGATION-API]
```

For instance, see how the [Fetch](https://fetch.spec.whatwg.org/#websocket-protocol) spec refers to WebSockets.

The `<cite>` tag doesn't seem to get autolinked, but it's rendered in italics. Bikeshed does, however, linkify the part in square brackets. The Bikeshed docs say that `[[FOO]]` is for normative references and `[[!FOO]]` is for informative references, so I suppose the context matters there.

Reply:
I should call out that I've now made some editorial decisions that enabled me to avoid explicitly linking to Private Aggregation's spec. I think it's actually a little clearer this way, too, because we autolink to the relevant dfns instead of dropping the reader on the front door of the Private Aggregation spec.